clang/lib/CIR/CodeGen/CIRGenCoroutine.cpp

```diff
@@ -194,11 +194,11 @@ CIRGenFunction::buildCoroAllocBuiltinCall(mlir::Location loc) {
 
   mlir::cir::FuncOp fnOp;
   if (!builtin) {
-    fnOp =
-        CGM.createCIRFunction(loc, CGM.builtinCoroAlloc,
-                              builder.getFunctionType(mlir::TypeRange{int32Ty},
-                                                      mlir::TypeRange{boolTy}),
-                              /*FD=*/nullptr);
+    fnOp = CGM.createCIRFunction(
+        loc, CGM.builtinCoroAlloc,
+        builder.getType<mlir::cir::FuncType>(mlir::TypeRange{int32Ty},
+                                             mlir::TypeRange{boolTy}, false),
```

Comment:
Please annotate literals with the argument name, e.g. `... /*isVariadic=*/false` (Replace isVariadic for the actual name used in the param description)